_data/simple-icons.json

```
@@ -7890,6 +7890,11 @@
             "hex": "290B53",
             "source": "https://cncf-branding.netlify.app/projects/kuma/"
         },
+        {
+            "title": "Kununu",
+            "hex": "FFC62E",
+            "source": "https://news.kununu.com/presseinformationen"
```

```suggestion
            "source": "https://www.kununu.com"
```
Given we're using the icon version from the homepage, we should link that as our source. I would also argue the press area doesn't provide any guidelines on how the icon should be used - so shouldn't be included.